packages/front-end/pages/metric/[mid].tsx

```diff
@@ -617,40 +622,78 @@ const MetricPage: FC = () => {
                                 Over Time
                               </h5>
                             </div>
-                            <div className="col-auto">
-                              <a
-                                className={clsx("badge badge-pill mr-2", {
-                                  "badge-light": groupby === "week",
-                                  "badge-primary": groupby === "day",
-                                })}
-                                href="#"
-                                onClick={(e) => {
-                                  e.preventDefault();
-                                  setGroupby("day");
-                                }}
-                              >
-                                day
-                              </a>
-                              <a
-                                className={clsx("badge badge-pill", {
-                                  "badge-light": groupby === "day",
-                                  "badge-primary": groupby === "week",
-                                })}
-                                href="#"
-                                onClick={(e) => {
-                                  e.preventDefault();
-                                  setGroupby("week");
-                                }}
-                              >
-                                week
-                              </a>
+                          </div>
+
+                          <div className="row mt-4 mb-1">
+                            <div className="col">
+                              <strong className="ml-4 align-bottom">
+                                Daily Average
```

Comment:
Could adding a ? tooltip here be too cluttered? I'd love to add the following text somewhere on this page, rather than buried in the docs (although maybe the Metric page docs are the right place, with just a link in the tooltip or something).

It could be next to the daily average and read:

"the figure shows the total metric value on a day divided by number of unique units (e.g. users) in the metric source on that day. {The standard deviation shows the spread of the daily user metric values}. When, smoothing is turned on, we simply average values over the 7 trailing days (including the selected day)."

we could just show the part in `{}` when the metric is not binomial (e.g. when the stddev is showing)